associated-token-account/program/Cargo.toml

```diff
@@ -21,6 +21,7 @@ spl-token = { version = "0.2", path = "../../token/program-2022", package = "spl
 thiserror = "1.0"
 
 [dev-dependencies]
+spl-token-legacy = { version = "3.3", path = "../../token/program", package = "spl-token", features = ["no-entrypoint"] }
```

Comment:
I vote for calling spl-token spl-token, and spl-token-2022 spl-token-2022 myself.  Which would mean this line changes to `spl-token` and the import of token 2022 higher up changes to `spl-token-2022` (as well as in the ATA code)

Reply:
Done everywhere, that makes sense